packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -1345,13 +1345,6 @@ describe( 'Navigation', () => {
 			await page.waitForXPath(
 				`//*[contains(@class, 'components-snackbar__content')][ text()="You do not have permission to edit this Menu. Any changes made will not be saved." ]`
 			);
-
-			// Expect a console 403 for request to Navigation Areas for lower permission users.
-			// This is because reading requires the `edit_theme_options` capability
-			// which the Contributor level user does not have.
-			// See: https://github.com/WordPress/gutenberg/blob/4cedaf0c4abb0aeac4bfd4289d63e9889efe9733/lib/class-wp-rest-block-navigation-areas-controller.php#L81-L91.
-			// Todo: removed once Nav Areas are removed from the Gutenberg Plugin.
-			expect( console ).toHaveErrored();
```

Comment:
🥳

Reply:
Actually, we can't remove it as there are still quite a few 403 errors in the console:

![CleanShot 2022-05-24 at 11 50 26](https://user-images.githubusercontent.com/205419/170004025-786381f7-b27d-4a93-9628-5e932660b4ad.png)

I'll just update the comment and leave the expect() in